### ISSUE TITLE
[Bug] 初始化ListTable，columns不传或传空数组，报错

### ISSUE THREAD

**Issue by Char-Ten** [VisActor/VTable#405]
### Version

0.12.0

### Link to Minimal Reproduction

null

### Steps to Reproduce

<img width="427" alt="image" src="https://github.com/VisActor/VTable/assets/17474452/7a10049a-11af-473f-afb4-187a32ba84fb">
<img width="955" alt="image" src="https://github.com/VisActor/VTable/assets/17474452/38821853-4b37-4e0a-8fab-87c73eaeb471">
<img width="718" alt="image" src="https://github.com/VisActor/VTable/assets/17474452/4273bea6-b444-4451-ad65-c826a153a148">

报错位置:ListTable.js:67:20

### Current Behavior

报错，页面空白

### Expected Behavior

是否加上`||{title:''}`可以解决？
```js
const {title: title} = table.internalProps.layoutMap.getHeader(col, row) || {title:''};
```

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by fangsmile** [comment]
我们先看下 ，这两天会解决

**Comment by fangsmile** [comment]
resolved 0.12.2
